.github/dependabot.yml

```diff
@@ -21,3 +21,10 @@ updates:
       # Releases too often, it's annoying
       - dependency-name: "org.assertj:*"
         update-types: ["version-update:semver-patch"]
+  - package-ecosystem: docker
+    directory: "/src/main/docker"
+    schedule:
+      interval: weekly
+    ignore:
+      - dependency-name: "*"
+        update-types: [ "version-update:semver-major" ]
```

Comment:
so no need to specify the docker repository like we've done for Search ? nice! 👍🏻

Reply:
go go go 😄